src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -214,34 +219,37 @@
      * otherwise - {@code false}
      */
     public Endpoint(
-            String id,
-            Conference conference,
-            Logger parentLogger,
-            boolean iceControlling)
+        String id,
+        Conference conference,
+        Logger parentLogger,
+        boolean iceControlling,
+        Clock clock)
         throws IOException
     {
         super(conference, id, parentLogger);
 
+        this.clock = clock;
+
         creationTimeMillis = System.currentTimeMillis();
         diagnosticContext = conference.newDiagnosticContext();
-        transceiver
-                = new Transceiver(
-                id,
-                TaskPools.CPU_POOL,
-                TaskPools.CPU_POOL,
-                TaskPools.SCHEDULED_POOL,
-                diagnosticContext,
-                logger,
-                Clock.systemUTC());
+        transceiver = new Transceiver(
+            id,
+            TaskPools.CPU_POOL,
+            TaskPools.CPU_POOL,
+            TaskPools.SCHEDULED_POOL,
+            diagnosticContext,
+            logger,
+            Clock.systemUTC()
```

Comment:
Presumably you should now pass `clock` to `Transceiver` rather than using `Clock.systemUTC()` explicitly?